x/subscription/spec/02_state.md

```diff
@@ -0,0 +1,35 @@
+<!--
+order: 2
+-->
+
+# State
+
+The `x/subscription` stores the plans and subscriptions on chain:
+
+```golang
+type Plan struct {
+  id int,  // auto-increasing unique identifier
+  title string,
+  description string,
+  owner Address,  // beneficial owner of the plan
+  price Decimal,  // price in fiat currency show to user
+  fiat_currency String,  // fiat currency name
+  token_denom String,  // denomation of token
+  conversion_rate Decimal,  // conversion rate from fiat currency to token, maintained by owner
+
+  interval int, // number of seconds of payment period
+  timezone Timezone, // the timezone used to offset the time interval
+  last_collection_period int, // period index = (timestamp + timezone_offset) / interval
```

Comment:
probably good for @christopher-crypto to comment on this one -- is ok to collect on the plan-level, or on the subscriber-level (given subscriptions start at different times)?

Reply:
@christopher-crypto : real world may need both; but ok to start with the set interval

Reply:
stick to unified plan level collection time for now.